src/test/groovy/com/bmuschko/gradle/docker/internal/RegistryAuthLocatorTest.groovy

```diff
@@ -152,6 +152,29 @@ class RegistryAuthLocatorTest extends Specification {
         0 * logger.error(*_)
     }
 
+    def "AuthLocator returns correct default registry"() {
+        given:
+        String image = 'ubuntu'
+
+        when:
+        String registry = RegistryAuthLocator.getRegistry(image)
+
+        then:
+        registry == 'https://index.docker.io/v1/'
+    }
+
+
+    def "AuthLocator returns correct custom registry"() {
+        given:
+        String image = 'gcr.io/distroless/java17'
+
+        when:
+        String registry = RegistryAuthLocator.getRegistry(image)
```

Comment:
Shouldn't use the static method here. Use the instance instead.

Reply:
Done